cypress/e2e/awx/resources/workflowJobTemplates.cy.ts

```diff
@@ -147,56 +145,36 @@ describe('Workflow Job templates form', () => {
           this.globalProject as Project
         ).then((projectNode) => {
           cy.createAwxWorkflowVisualizerJobTemplateNode(workflowJobTemplate, jobTemplate).then(
-            (jobTemplateNode) => {
```

Comment:
These lines were removed because they felt like unnecessary complexity in this test....adding nodes and links via the api to a workflow visualizer, for the purpose of ensuring the job is launch and renders the output properly.  Removing these lines simplifies and speeds up the test.